cypress/integration/queryBuilder.spec.js

```diff
@@ -44,6 +44,29 @@ describe('test query builder', () => {
     });
   });
 
+  describe('timestamps are displayed as expected', () => {
+    beforeEach(() => {
+      cy.get('[data-cy=selectTable]').click();
+      cy.get('[data-cy=menuItem-all_data_types]').click();
+    });
+
+    it('correctly displays timestamps', () => {
+      // Click the header to make sure we don't sort or trigger an error
+      cy.get('[data-cy=columnHeader-timestamp_column]').scrollIntoView().click({ force: true });
+
+      cy.get('[data-cy=cellValue-timestamp_column-0]').should('have.text', '5/31/2023, 9:02:40 PM');
```

Comment:
Sweet!  I don't know if it's necessarily worth it but I wonder if we might ever want to experiment with mocking browser info to see what happens when it's set to, say, display dates in `DD/MM/YYYY`.  Note: this isn't a thing I think we should do for this particular PR but just wanted to raise a potential thing we might want to do